docs/pages/versions/unversioned/sdk/app-integrity.mdx

```diff
@@ -0,0 +1,186 @@
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-app-integrity'
+packageName: '@expo/app-integrity'
+platforms: ['ios', 'android']
+iconUrl: '/static/images/packages/expo-maps.png'
+isAlpha: true
+---
+
+import APISection from '~/components/plugins/APISection';
+import { APIInstallSection } from '~/components/plugins/InstallSection';
+import { Collapsible } from '~/ui/components/Collapsible';
+import { Terminal } from '~/ui/components/Snippet';
+import { CODE } from '~/ui/components/Text';
+import { ConfigReactNative } from '~/ui/components/ConfigSection';
+
+`@expo/app-integrity` provides APIs to help ensure your backend resources are accessed only by legitimate installations of your app running on genuine devices. It uses Apple's [App Attest service](https://developer.apple.com/documentation/devicecheck/establishing-your-app-s-integrity) on iOS and Google's [Play Integrity APIs](https://developer.android.com/google/play/integrity) on Android to verify app authenticity, helping prevent unauthorized clients, modified apps, or automated scripts from making requests to your server.
+
+In simple terms, `@expo/app-integrity` helps your server tell the difference between:
+
+- Your **real app** running on a **real device**, and
+
+- Anything else (modified apps, scripts, emulators).
+
+It does this by using the platform-recommended app attestation services.
+
+## Installation
+
+<APIInstallSection />
+
+## Usage on iOS
+
+### Configuration
+
+In Xcode, go to **Signing & Capabilities**, click **+ Capability**, add **App Attest**. Xcode will automatically add the required entitlement to your app.
+
+> **Note** To use the App Attest service, your app must have an App ID that you register on the Apple Developer website.
+
+For verification logic on your server, see [Validating apps that connect to your server](https://developer.apple.com/documentation/devicecheck/validating-apps-that-connect-to-your-server).
+
+### Check if the device supports App Attestation
+
+Not all devices can use the App Attest service, so it’s important to have your app run a compatibility check before accessing the service. If the user’s app doesn’t pass the compatibility check, gracefully bypass the service. You check for availability by reading the `isSupported` property.
+
+```js
+import * as AppIntegrity from '@expo/app-integrity';
+
+if (AppIntegrity.isSupported) {
+  // Perform key generation and attestation.
+}
+// Continue with your server API access.
+```
+
+> **Note** App Attest is not supported on simulator.
+
+> **important**  Most app extensions don’t support App Attest. Generally, when executing code in these extensions, bypass key generation and attestation, even if the `isSupported` method property is `true`. The only app extensions that support App Attest are watchOS extensions in watchOS 9 or later. For these extensions, you can use the results from `isSupported` to indicate whether your WatchKit extension bypasses attestation.
```

Comment:
```suggestion
> **info**  Most app extensions don’t support App Attest. Generally, when executing code in these extensions, bypass key generation and attestation, even if the `isSupported` method property is `true`. The only app extensions that support App Attest are watchOS extensions in watchOS 9 or later. For these extensions, you can use the results from `isSupported` to indicate whether your WatchKit extension bypasses attestation.
```

Let's change the callout type to info instead of important since we use important mostly for experimental features.